docs/paper.md

```diff
@@ -55,10 +55,10 @@ In addition, ASTE provides performance and accuracy metrics of the configured si
 Besides preCICE itself, core ingredients for practical applications are preCICE API language bindings, preCICE adapters, the simulation frameworks and their dependencies.
 ASTE, on the other hand, replaces coupled models and only requires a reduced set of dependencies.
 It abstracts the computational complexity of the models away by extracting the relevant information from VTK files instead and passing extracted data to preCICE, potentially in parallel on distributed memory.
-While the VTK files may stem from actual simulations, ASTE can also generate artificial VTK files with prescribed coupling data.
+These VTK files can be artificially generated or imported from an existing preCICE simulation.
```

Comment:
I think the sentence is at the same time too concrete and too vague.
I propose to reduce this to "Make or import them". Everything concrete is handled in the section below.

> While the VTK files may stem from actual simulations

The use-case is to study an existing scenario from a preCICE simulation. So lets mention this.

> ASTE can also generate artificial VTK files with prescribed coupling data.

Tools and data are covered in the section below. Let's leave them out.

Reply:
> ASTE can also generate artificial VTK files

Hm, the capabilities of ASTE being able to generate those files is now implicitly hidden in the python script links, right?

Reply:
ASTE is the testing environment which contains the scripts etc.
Generation is part of that.
Theoretically the mapping-tester too.

Reply:
>  Generation is part of that.

I agree, the discussion here is about where we write this in the paper.

Reply:
> Hm, the capabilities of ASTE being able to generate those files is now implicitly hidden in the python script links, right?

Ah sorry, I missed the "link" part.

Lets clarify that these scripts are actually included in ASTE:

> The VTK files can be generated from mesh generation tools (e.g., GMSH [@gmsh]), included [Python scripts](https://github.com/precice/aste/tree/develop/tools/mesh-generators), other simulation software, or directly reused from a [completed preCICE simulation](https://precice.org/configuration-export.html).

Reply:
The VTK files can be imported from an existing preCICE simulation? This doesn't sound right. If anything, they can be 'reused from a completed simulation".

I don't like the proposed change in this line, though. There is value in clearly stating that "ASTE can generate VTK files with prescribed coupling data". The use case is a need in our application and the statement is not equivalent to "VTK files can be generated from included Python scripts".

I would go for the two changes below then if you agree.

Reply:
Ok. Let's do that

Reply:
```suggestion
While the VTK files may stem from actual simulations, ASTE can also generate artificial VTK files with prescribed coupling data.
```